users/models.py

```diff
@@ -43,8 +43,8 @@ class CustomUser(AbstractUser):
                                  default='GST')
     linkedin = models.URLField(max_length=64, null=True, blank=True)
     facebook = models.URLField(max_length=64, null=True, blank=True)
-    created_at = models.DateTimeField(auto_now=True, editable=False)
-    modified_at = models.DateTimeField(auto_now_add=True, editable=False)
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    modified_at = models.DateTimeField(auto_now=True, editable=False)
```

Comment:
I think migrations are to be made for this particular change. (I'm ofcourse not retaining `0005_auto_20210116_0937.py` in my local, showing the statement below just to explain).
```
Migrations for 'users':
  users/migrations/0005_auto_20210116_0937.py
    - Alter field created_at on customuser
    - Alter field modified_at on customuser
```

Reply:
I forgot 😝 Thanks for reminding 🙌

Reply:
No problem 😂